test/smoke_test/smoke_test.sh

```diff
@@ -33,6 +33,9 @@ do
         testFile=bulk_data_requests_lite.sh
         if [ ${CMS_ID} = "A9996" ]; then
                 testFile=bulk_data_requests.sh
+        elif [ ${CMS_ID} = "A9997" ]; then
+                docker-compose exec -T db sh -c 'psql "postgres://postgres:toor@db:5432/bcda?sslmode=disable" -c "UPDATE acos SET blacklisted=true WHERE cms_id = '"'"${CMS_ID}"'"'"'
```

Comment:
Why don't we use the new CLI instead?

Reply:
I wanted to decouple the two PRs, but I think I can merge the CLI one first and then update the script to use the command.

Reply:
This is no longer needed since the negative tests are being verified in the postman test suite.